go/sqltypes/proto3_test.go

```diff
@@ -35,6 +35,13 @@ func TestResult(t *testing.T) {
 			NULL,
 			NULL,
 		}},
+		Extras: &querypb.ResultExtras{
```

Comment:
We're gonna add more to it, at least the freshness bool. See the design doc. It's a nice parallel to ExecuteOptions too.